packages/components/src/DateTimePickers/ViewHeader/ViewHeader.component.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import theme from './ViewHeader.scss';
+
+function ViewHeader(props) {
```

Comment:
I think you should put that directly into ViewLayout. No need to split every part. That makes
* the component too tiny
* more component, more file, more difficult to navigate in code
* more extra div layers

Just set the 3 divs with its simple style in ViewLayout header div please

Reply:
&#10003;